app/unit-tests/resources/commcare-apps/update_info_form/default/app_strings.txt

```diff
@@ -0,0 +1,29 @@
+app.display.name=Update Info Form
```

Comment:
In the interest of keeping test resources as concise as possible, you should be able to delete both `app_strings` files.

Reply:
Ah yea you're right, I mis-remembered. Sorry about that.